### ISSUE TITLE
Zero token balance after switching to 7.4.0 branch.

### ISSUE THREAD

**Issue by vi** [Tribler/tribler#5006]
##### Tribler version/branch+revision: #####

v7.4.0-exp3

##### Operating system and version: #####

Linux

##### Steps to reproduce the behavior: #####

Run Tribler 7.3.2. Then hiatus. Then switch to the v7.4.0 branch.

##### Expected behavior: #####

Token balance stays approximately the same as was left before.

##### Actual behavior: #####

"0 MBytes given to community, 0 MBytes taken from community".

##### Relevant log file output: #####

This gets logged repeatedly:

```
ERROR   1576450617.34                 community:695  (TriblerTunnelCommunity)  Exception occurred while handling packet!
Traceback (most recent call last):
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/community.py", line 692, in on_packet_from_circuit
    self.decode_map_private[msg_id](source_address, data, circuit_id)
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/community.py", line 76, in wrapper
    return inner_wrapper(self, source_address, data)
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/lazy_community.py", line 104, in wrapper
    return func(self, source_address, *unpacked)
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/community.py", line 74, in inner_wrapper
    return func(inner_self, inner_source_address, *combo)
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/community.py", line 774, in on_created
    self._ours_on_created_extended(circuit, payload)
  File "/home/tribler/tribler/Tribler/community/triblertunnel/community.py", line 447, in _ours_on_created_extended
    super(TriblerTunnelCommunity, self)._ours_on_created_extended(circuit, payload)
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/community.py", line 551, in _ours_on_created_extended
    payload.auth, hop.public_key.key.pk)
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/tunnelcrypto.py", line 51, in verify_and_generate_shared_secret
    libnacl.crypto_auth_verify(auth, dh_received, shared_secret)
  File "/home/tribler/.local/lib/python3.7/site-packages/libnacl/__init__.py", line 887, in crypto_auth_verify
    raise ValueError('Invalid secret key')
ValueError: Invalid secret key
```

**Comment by devos50** [comment]
The error was caused by an error in the upgrade logic. This is most likely fixed by #5093. We will release Tribler 7.4.1 as soon as possible 👍 .

**Comment by Dmole** [comment]
This got worse from 7.4.1 to 7.4.2.
